### DISCUSSION TITLE
Adding a new directive

### DISCUSSION THREAD

**Discussion by sbellem** [executablebooks/meta#521] [Q&A]
Hi, I am wondering whether it would be possible to add a new directive, or to support defining custom directives. The directive I am currently interested in is `protocol`. I am not sure if that would be welcome, but if yes, I would be happy to help adding it.

**Comment by michaelosthege** [comment]
I believe this should be covered by https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#directives-a-block-level-extension-point
